tests/test_data_base.py

```diff
@@ -0,0 +1,77 @@
+import psycopg2
+import pytest
+from psycopg2 import sql
+
+from src.data_base import DataBase
+
+
+@pytest.fixture
+def table_name():
+    return "test_data_base"
```

Comment:
This should be a constant, not a fixture

Reply:
Done